Templates/Swift/ResponseType.stencil

```diff
@@ -25,8 +25,10 @@
 		{{ accessLevel }} var as{{ fragment.name }}Fragment: {{ moduleName }}.{{ fragment.name }}{% if fragment.conditionallySelected %}?{% endif %}
 	{% endfor %}
 
+	{% if graphQLName != "Subscription" %}
```

Comment:
We shouldn't need to do this. 

With the changes done in https://github.com/Shopify/syrup/pull/22 `typename` can and should be handled like any other field.

Reply:
I was hoping so as well. `renderResponseTypes` uses collected fields whereas `renderOperation` uses selection set. I have always been confused about the distinction between these

Reply:
Currently the response templates don't get access to the selection set we updated in that PR. We could pass the selection set into the method but would that be equivalent?

Reply:
Do you recall why even though `__typename` is implicitly added to queries that it isn't represented in the Kotlin `Response` models? Seems odd to me even if I explicitly query for it, there isn't a property generated for it. It's only used in generating our Selections. Swift code generates a property but it looks like it's only used internally. 🤔